lib/install/management/store_files.go

```diff
@@ -126,7 +126,7 @@ func (d *Dispatcher) deleteDatastoreFiles(ds *object.Datastore, path string, for
 	}
 
 	m := object.NewFileManager(ds.Client())
-	if d.isVSAN(ds) {
+	if d.geVC65() || d.isVSAN(ds) {
```

Comment:
the pkg/vsphere/datastore is meant to be a common pkg could be used by both vic-machine and storage portlayer, but there is some assumption not exists in vic-machine.
I could refactor code to merge them two, but probably not at this time.